src/components/Expandable.tsx

```diff
@@ -5,6 +5,7 @@ import 'components/styles/Expandable.scss'
 
 export const ControlledExpandable: React.FunctionComponent<{
   id: string
+  noContract?: boolean
```

Comment:
IIUC, the way you are using this is to make an object that normally can be opened or closed via clicking not work. 

I think it is better for user expectations if all icons always work a certain way (i.e. all chevrons are clickable); could we use a different component for "thing that is expanded and cannot be clicked to collapse?"

Reply:
Oh, also this leaves the chevron + header with a different cursor when mousing over, which is misleading.

Reply:
Better now? Made the icon flat when it's not contractable.

Also separated out the stuff that shows up in manual entry mode vs regular old details and as a happy side effect got rid of the "remember last location mode details open state" logic.